app/server/services/setupSecrets.ts

```diff
@@ -0,0 +1,22 @@
+import * as config from 'config';
+import * as propertiesVolume from '@hmcts/properties-volume';
+import { get, set } from 'lodash';
+
+propertiesVolume.addTo(config);
+
+function setSecret (secretPath: string, configPath: string) {
```

Comment:
normally you should always specify a type for the return value
```suggestion
function setSecret (secretPath: string, configPath: string): void {
```